examples/damsvalley.jl

```diff
@@ -170,7 +170,7 @@ end
 function solve_dams(display=false)
     model, params = init_problem()
 
-    V, pbs = optimize(model, params, display)
+    V, pbs = solve_SDDP(model, params, display)
```

Comment:
We have to set a reunion to brainstorm about the naming. Glad you bring the subject with this modification